### ISSUE TITLE
"Cannot call method 'trim' of null" after submitting pull request

### ISSUE THREAD

**Issue by dantewang** [node-gh/gh#363]
I have following global configs in .gitconfig:

```
[user]
        name = Dante Wang
        email = dantewang@example.com
```

while submitting pull request, I got the following error (after pull request successfully submitted):

```
"Dante Wang"

/usr/lib/node_modules/gh/lib/git.js:64
        if (myUser === user.trim()) {
                            ^
TypeError: Cannot call method 'trim' of null
    at ChildProcess.<anonymous> (/usr/lib/node_modules/gh/lib/git.js:64:29)
    at ChildProcess.emit (events.js:98:17)
    at maybeClose (child_process.js:756:16)
    at Process.ChildProcess._handle.onexit (child_process.js:823:5)
```

**Comment by zenorocha** [comment]
I just released a [new beta version](https://github.com/node-gh/gh/releases/tag/v2.0.0-alpha) which should fix this problem.

**Attention:** In order to use it, you'll need NodeJS `0.12.0`.

Let me know if you still face problems.

**Comment by dantewang** [comment]
On nodejs 0.12.0,

with gh 2.0.0 alpha

```
sudo npm install gh -g
/bin/gh -> /lib/node_modules/gh/bin/gh.js
gh@2.0.0-alpha /lib/node_modules/gh
├── cli-log@0.0.8
├── truncate@1.0.4
├── which@1.0.8
├── userhome@1.0.0
├── open@0.0.5
├── wordwrap@0.0.2
├── async@0.9.0
├── github@0.2.3
├── moment@2.9.0
├── nopt@3.0.1 (abbrev@1.0.5)
├── handlebars@3.0.0 (optimist@0.6.1, uglify-js@2.3.6, source-map@0.1.43)
├── cli-color@0.3.2 (d@0.1.1, timers-ext@0.1.0, memoizee@0.3.8, es5-ext@0.10.6)
├── inquirer@0.8.0 (ansi-regex@1.1.1, figures@1.3.5, mute-stream@0.0.4, through@2.3.6, readline2@0.1.1, chalk@0.5.1, lodash@2.4.1, rx@2.4.1)
├── update-notifier@0.3.1 (is-npm@1.0.0, string-length@1.0.0, semver-diff@2.0.0, chalk@1.0.0, latest-version@1.0.0, configstore@0.3.2)
└── insight@0.5.3 (object-assign@2.0.0, lodash.debounce@3.0.2, tough-cookie@0.12.1, chalk@1.0.0, os-name@1.0.3, request@2.53.0, configstore@0.3.2)
```

still having the issue.

**Comment by zenorocha** [comment]
Hey @dantewang, there was no errors in this log. Could you please provide a more extensive log?

**Comment by zenorocha** [comment]
Problem detected, this error was caused after: https://github.com/node-gh/gh/commit/af9e01dade6905f9524de55983c7a5bcf7a402ee

**Comment by zenorocha** [comment]
@mdelapenya also reported this problem.

**Comment by zenorocha** [comment]
It should be fixed now, please update gh: `npm install gh -g`.

If you're using any plugins there's still one bug to be fixed #364, we decided to release this version to stop blocking you guys.

**Comment by dantewang** [comment]
@zenorocha confirmed to be working now. However, another problem after closing the pr in hook:

```
 [hook] gh pr 11 --close
 [warn] Unparsable response. SyntaxError: Unexpected token C
/usr/lib/node_modules/gh/lib/logger.js:80
    if (msg.errors && msg.errors[0] && msg.errors[0].message) {
           ^
TypeError: Cannot read property 'errors' of undefined
    at Object.logger.getErrorMessage (/usr/lib/node_modules/gh/lib/logger.js:80:12)
    at /usr/lib/node_modules/gh/lib/hooks.js:49:68
    at ChildProcess.exithandler (child_process.js:751:5)
    at ChildProcess.emit (events.js:110:17)
    at maybeClose (child_process.js:1008:16)
    at Process.ChildProcess._handle.onexit (child_process.js:1080:5)
```

Any idea or should I report a new issue?
